omnibus/config/software/habitat.rb

```diff
@@ -0,0 +1,38 @@
+name "habitat"
+default_version "master"
+license "Apache-2.0"
```

Comment:
The source code is apache licensed, but the binary is not. The binary uses Chef's MLSA.

Reply:
It looks like "Chef EULA" is the right one from the description: https://docs.chef.io/chef_license/#chef-eula